TestProd/WebContent/login.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="US-ASCII">
+<title>Login Page</title>
+<script src="https://www.google.com/recaptcha/api.js"></script>
+</head>
+<body>
+	<form action="LoginServlet" method="post">
+		Username: <input type="text" name="user"> <br>
+		Password: <input type="password" name="pwd"> <br>
+		<div class="g-recaptcha" data-sitekey="6LfWtx0UAAAAAPBRQz2ociklfzJrolyzfCt8KlK2"></div>
```

Comment:
Not a good idea to directly put data-sitekey here. Read it from somewhere (may be a property file). Not sure how it is handled in UI.